src/io/PipeWriter.zig

```diff
@@ -1157,8 +1157,9 @@ pub fn WindowsStreamingWriter(
             this.source = null;
             if (this.closed_without_reporting) {
                 this.closed_without_reporting = false;
-                onClose(this.parent);
```

Comment:
pretty sure that this will cause issues, lets see CI, my guess is streams and IPC